source/_integrations/history_stats.markdown

```diff
@@ -121,7 +121,11 @@ Depending on the sensor type you choose, the `history_stats` integration can sho
 
 - **time**: The default value, which is the tracked time, in hours
 - **ratio**: The tracked time divided by the length of your period, as a percentage
-- **count**: How many times the tracked entity matched the configured state during the time period
+- **count**: How many times the tracked entity matched the configured state during the time period. Unlike a rising edge counter, this will also include a count if the entity is already in the matched state at the beginning of the time period.
```

Comment:
I'm not a microcontroller developer and I'm not sure rising edge counter is a commonly understood term.
The current description states that the counter only increases if the actual state is matched within the time period. It doesn't say anywhere that we're counting falling or rising edges. So I'm not sure why one would assume we're counting edges here. To me, this addition is confusing. Maybe you can add an example instead that illustrate how it counts?

Reply:
Yes I'll think about how best to put an example here. For wording, instead of `rising edge` I could use `event` but that word means something else in HA. Perhaps:
> Unlike counting transitions to the state, this will also include a count if the entity is already in the matched state at the beginning of the time period

I could then show a history graph of a light that was already 'on' at the start of the window, and show that the states counter would calculate e.g. 4 while the number of times the light got _turned on_ during the window was only e.g. 3. 

The point of all this is that, quite often, people want to know how many times something was turned on or turned off, and this `count` feature in history_stats doesn't actually count that. Instead, this feature counts how many times something was in that state. The fact that there is a difference between those two counting methods isn't obvious unless it's pointed out.

Reply:
Thank you for clarifying. I find your explanation very helpful.

Reply:
I didn't include a history graph image but hopefully the new text is clear enough.